content/hardware/03.nano/carriers/nano-screw-terminal-adapter/datasheet/datasheet.md

```diff
@@ -188,3 +188,4 @@ Hereby, Arduino S.r.l. declares that this product is in compliance with essentia
 | Date       | **Revision** | **Changes**               |
 | ---------- | ------------ | ------------------------- |
 | 17/06/2022 | 1            | First release             |
+| 21/01/2025 | 2            | Fixed board dimensions    |

```

Comment:
hello @pedromsousalima please remember **we put the order in the opposite way, being the latest revision version the first one, therefore:**

```suggestion
```